app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,78 @@
+class RecipesController < ApplicationController
```

Comment:
- [ ] [OPTIONAL] When I run the program the first time I wasn't able to run it because the app is not defined as a Rails app as the bin folder is deleted, Kindly run the command:
```
rake app:update:bin
```
to fix the problem. This command fixes the problem on my side.